virttest/virt_vm.py

```diff
@@ -638,6 +638,7 @@ def get_address(self, index=0):
         :raise VMAddressVerificationError: If the MAC-IP address mapping cannot
                 be verified (using arping)
         """
+
         nic = self.virtnet[index]
         self.ip_version = self.params.get("ip_version", "ipv4").lower()
```

Comment:
@sathnaga 
My fault, guess I mess it up. I didn't notice that when add a new param here, it will cause a lot of changes, and for new caller, we always need pass it explicit. Maybe we can reopen the original one, what do you think? Sorry for make you confused